DataManagementSystem/Service/FileCatalogHandler.py

```diff
@@ -142,12 +142,13 @@ def export_getPathPermissions( self, lfns ):
     return gFileCatalogDB.getPathPermissions( lfns, self.getRemoteCredentials() )
   
   
-  types_hasAccess = [StringTypes, [ ListType, DictType ] + list( StringTypes ) ]
-  def export_hasAccess( self, opType, paths ):
+  types_hasAccess = [ dict, basestring ]
```

Comment:
This is the point, with the updated client it will be always dict as the checkArgumentFormat is now done on the client side